src/Events/Integrations/Plugins/Tickets_Wallet_Plus/Apple_Wallet/Event_Data_Add_Info.php

```diff
@@ -0,0 +1,106 @@
+<?php
```

Comment:
Just an FYI. We're changing the structure just a little bit. We're adding a `Passes` folder within `Tickets_Wallet_Plus`.